### PR TITLE
sync code block example with dom-examples

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

The code block example was using different one from dom-examples, I corrected and modified to match what we discuss in this document.

### CODE REVIEW

File: files/en-us/web/api/response/url/index.md [hunk 1]
```diff
@@ -27,13 +27,15 @@ const myImage = document.querySelector("img");
 
 const myRequest = new Request("flowers.jpg");
 
-fetch(myRequest).then((response) => {
-  console.log(response.url); // returns https://developer.mozilla.org/en-US/docs/Web/API/Response/flowers.jpg
-  response.blob().then((myBlob) => {
+fetch(myRequest)
+  .then((response) => {
+    console.log('response.url =', response.url); // response.url is "https://developer.mozilla.org/en-US/docs/Web/API/Response/flowers.jpg"
```
Comment (yin1999) [inline files/en-us/web/api/response/url/index.md:32]: Should we use the output of the live sample

image

```suggestion
    console.log('response.url =', response.url); // response.url = https://mdn.github.io/dom-examples/fetch/fetch-response/flowers.jpg
```

File: files/en-us/web/api/response/url/index.md [hunk 2]
```diff
@@ -27,13 +27,15 @@ const myImage = document.querySelector("img");
...
+    return response.blob();
+  })
+  .then((myBlob) => {
     const objectURL = URL.createObjectURL(myBlob);
     myImage.src = objectURL;
-  });
-});
+  })
```
Comment (yin1999) [inline files/en-us/web/api/response/url/index.md:38]: maybe we still need the tailing `;` (prettier would add this)

```suggestion
  });
```

File: files/en-us/web/api/response/status/index.md [hunk 3]
```diff
@@ -30,13 +30,15 @@ const myImage = document.querySelector("img");
 
 const myRequest = new Request("flowers.jpg");
 
-fetch(myRequest).then((response) => {
-  console.log(response.status); // returns 200
-  response.blob().then((myBlob) => {
+fetch(myRequest)
+  .then((response) => {
+    console.log('response.status =', response.status); // response.status is 200 if requested successfully
```
Comment (jasonren0403) [inline files/en-us/web/api/response/status/index.md:35]:
```suggestion
    console.log('response.status =', response.status); // response.status = 200
```

File: files/en-us/web/api/response/statustext/index.md [hunk 4]
```diff
@@ -32,13 +32,15 @@ const myImage = document.querySelector("img");
 
 const myRequest = new Request("flowers.jpg");
 
-fetch(myRequest).then((response) => {
-  console.log(response.statusText); // returns "OK" if the response returned successfully
-  response.blob().then((myBlob) => {
+fetch(myRequest)
+  .then((response) => {
+    console.log('response.statusText =', response.statusText); // response.statusText is "OK" if the response returned successfully
```
Comment (jasonren0403) [inline files/en-us/web/api/response/statustext/index.md:37]:
```suggestion
    console.log('response.statusText =', response.statusText); // response.statusText = "OK"
```

File: files/en-us/web/api/response/type/index.md [hunk 5]
```diff
@@ -40,13 +40,15 @@ const myImage = document.querySelector("img");
 
 const myRequest = new Request("flowers.jpg");
 
-fetch(myRequest).then((response) => {
-  console.log(response.type); // returns basic by default
-  response.blob().then((myBlob) => {
+fetch(myRequest)
+  .then((response) => {
+    console.log('response.type =', response.type); // response.type is 'basic' by default
```
Comment (jasonren0403) [inline files/en-us/web/api/response/type/index.md:45]:
```suggestion
    console.log('response.type =', response.type); // response.type = 'basic'
```

**Comment by jasonren0403** [comment]
Hi @sideshowbarker , the PR hasn't got any review for almost one month, could it get reviewed soon to avoid lagging too behind? Thanks 🙏

**Comment by sideshowbarker** [comment]
> Hi @sideshowbarker , the PR hasn't got any review for almost one month, could it get reviewed soon to avoid lagging too behind? Thanks 🙏

I see it got review from @yin1999 on August 6th and there’s still an unresolved comment at https://github.com/mdn/content/pull/28412#discussion_r1285078091 with a commit suggestion that you’ve not responded to?

**Review by sideshowbarker — APPROVED** [review]

**Comment by jasonren0403** [comment]
> > Hi @sideshowbarker , the PR hasn't got any review for almost one month, could it get reviewed soon to avoid lagging too behind? Thanks 🙏
> 
> I see it got review from @yin1999 on August 6th and there’s still an unresolved comment at [#28412 (comment)](https://github.com/mdn/content/pull/28412#discussion_r1285078091) with a commit suggestion that you’ve not responded to?

sorry for that, I have resolved them 🙏

**Pull request merged**